### DISCUSSION TITLE
Missing channels for EPG Guides

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#51] [Q&A] [answered]
Its working! Thanks!

One more thing. There is no EPG guides for those list:

AFV FAMILY*
AMC PRESENTS*
CONTV ANIME*
CRACKLE*
DOVE CHANNEL*
FAIL ARMY (RED BOX)*
FOX WEATHER LIVE *

**Comment by BellezaEmporium** [comment, marked as answer]
Please ask for those inside the [epg repository](https://github.com/iptv-org/epg)

**Comment by spotanjo3** [comment]
Ok!
